app/src/main/res/layout/question_player_fragment.xml

```diff
@@ -193,4 +194,4 @@
         android:layout_height="124dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
added